static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+          content: 
+            application/json:
+              schema: 
+                $ref: "#/components/schemas/validateTagsResponse"      
+components: 
+  schemas:
+    createFolderBody:
+      type: object
+      properties:
+        name:
+          type: string
+          example: SampleFolder
+        parentId:
+          type: string
+          example: 6a5e0927-1527-4abc-9993-376fd7067ca5
+    folder:
+      type: object
+      properties: 
+        id:
+          type: string
+          example: 83f8287c-767b-4106-b271-257282fd170e
+          description: The ID of the folder.
+        name:
+          type: string
+          example: SampleFolder
+          description: The name of the folder.
+        noun:
+          type: string
+          example: dataset
+          description: "The type of business objects that belong to the folder. Possible values include `dataset` and `segment`."
+        imsOrg:
+          type: string
+          example: '{ORG_ID}'
+          description: "The ID of the organization that the folder belongs to."
+        sandboxId:
+          type: string
+          example: "{SANDBOX_ID}"
+          description: "The ID of the sandbox that the folder belongs to."
+        sandboxName:
+          type: string
+          example: "prod"
+          description: "The name of the sandbox that the folder belongs to."
+        createdBy:
+          type: string
+          example: "{USER_ID}"
+          description: "The ID of the user who created the folder."
+        createdAt:
+          type: string
+          example: "2023-10-01T08:47:06.192+00:00"
+          description: "The date and time that the folder was created."
+        modifiedBy:
+          type: string
+          example: "{USER_ID}"
+          description: "The ID of the user who last modified the folder."
+        modifiedAt:
+          type: string
+          example: "2023-10-01T08:47:06.192+00:00"
+          description: "The date and time that the folder was last modified."
+        status:
+          type: string
+          example: "IN_USE"
+          description: "The status of the folder. Possible values include `IN_USE` and `ARCHIVED`."
+        _links:
+          type: object
+          properties:
+            self:
+              type: object
+              properties: 
+                href:
+                  type: string
+                  example: "/folders/dataset/83f8287c-767b-4106-b271-257282fd170e"
+          description: "A link to the folder."
+    foldersList:
+      type: object
+      description: "A list of subfolders for the requested folder."
+      properties: 
+        id:
+          type: string
+          example: "c626b4f7-223b-4486-8900-00c266e31dd1"
+          description: The ID of the folder.
+        name:
+          type: string
+          example: "ParentFolder"
+        noun:
+          type: string
+          example: "dataset"
+          description: The name of the folder.
```

Comment:
oh this is in the wrong place. 
Perhaps: "The type of business object." or something.